hack/helm/trento-dev/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: trento-server
   repository: file://../../../packaging/helm/trento-server
-  version: 0.3.8
-digest: sha256:296438dbc64a55fea561d037f5d6904a2599c88ff18ca2846bbccb2426bdfc6c
-generated: "2022-02-16T12:08:25.452419738+01:00"
+  version: 0.4.0
```

Comment:
hmm I don't think this is right, since you just bumped the version to 0.4.1. Maybe forgot to update the dev chart?

Reply:
Versions tend to misalign, I bumped all to the same version number some time ago but it's totally fine to have different versions for charts :/